go/oasis-node/cmd/common/metrics/mem.go

```diff
@@ -0,0 +1,98 @@
+// Package metrics implements a prometheus metrics service.
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
+)
+
+const (
+	MetricMemVmSizeBytes   = "oasis_worker_mem_VmSize_bytes" // nolint: golint
+	MetricMemRssAnonBytes  = "oasis_worker_mem_RssAnon_bytes"
+	MetricMemRssFileBytes  = "oasis_worker_mem_RssFile_bytes"
+	MetricMemRssShmemBytes = "oasis_worker_mem_RssShmem_bytes"
+)
+
+var (
+	vmSizeGauge   prometheus.Gauge // nolint: golint
```

Comment:
Why is `// nolint: golint` needed here?

Reply:
Removed. Before, it was `VmSizeGauge` member and `golint` complained that `Vm` Should be `VM` instead, but this contradicted the `/proc/<pid>/stats` naming then.